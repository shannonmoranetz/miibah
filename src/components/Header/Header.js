import React from 'react';

const Header = () => {
  return (
    <div className="Header">
      <img className="star" src={require(`../../styles/images/star2.png`)} alt="Star"/>
      <h1 className="title">miibah</h1>
      <img className="star" src={require(`../../styles/images/star2.png`)} alt="Star"/>
    </div>
  );
}

export default Header;
