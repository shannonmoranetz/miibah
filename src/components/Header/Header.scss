.Header {
  display: flex;
  justify-content: center;
  align-items: center;
}

.star {
  height: 100px;
  width: 100px;
  margin: 0px 30px;
}

.title {
  font-family: $heading-font;
  color: white;
  font-size: 80px;
}