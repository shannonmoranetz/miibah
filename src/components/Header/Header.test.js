import Header from './Header';

describe('Header', () => {});