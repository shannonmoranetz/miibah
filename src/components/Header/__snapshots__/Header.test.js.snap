// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<div
  className="Header"
>
  <img
    alt="Star"
    className="star star-left"
    src="star2.png"
  />
  <Link
    className="title"
    replace={false}
    to="/"
  >
    miibah
  </Link>
  <img
    alt="Star"
    className="star star-right"
    src="star2.png"
  />
</div>
`;
